dsp/window/window_example_test.go

```diff
@@ -0,0 +1,388 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
I'm not sure we need so many examples. We need at least one to show how they should be used and demonstrate the copying caveat, but apart from that they are all the same; we're not trying to teach DSP here.